plugins/translate.py

```diff
@@ -70,13 +70,13 @@ def lock_sliders(self):
 
     def on_apply(self):
 
+        approach = "relative" if self.relative.isChecked() else "absolute"
         if self.random.isChecked():
-            approach = random.choice(["relative", "absolute"])
-            x = random.seed(self.seed.getValue())
-            y = random.seed(self.seed.getValue())
-            z = random.seed(self.seed.getValue())
+            random.seed(self.seed.getValue())
+            x = random.random() * (self.seed.getValue() / 10)
+            y = random.random() * (self.seed.getValue() / 10)
+            z = random.random() * (self.seed.getValue() / 10)
```

Comment:
The  divide by 10 is quite arbitrary. Find a more elegant solution, lower the max of the seed slider?